metadata-suggestions.md

```diff
@@ -0,0 +1,528 @@
+              {
+                "value": "Smith",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "person.givenName": [
+              {
+                "value": "Dean",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ]
+        },
+        "type": "metadataSuggestionEntry",
+        "_links": {
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid/entryValues/0000-0002-4271-0436"
+          },
+          "changes": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid/entryValues/0000-0002-4271-0436/changes"
+          }
+        }
+      },
+      {
+        "id": "0000-0003-3681-2038",
+        "display": "Smith, Charles",
+        "value": "Smith, Charles",
+        "metadata": {
+            "dc.identifier.orcid": [
+              {
+                "value": "0000-0003-3681-2038",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "dc.identifier.uri": [
+              {
+                "value": "https://orcid.org/0000-0003-3681-2038",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "person.familyName": [
+              {
+                "value": "Smith",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ],
+            "person.givenName": [
+              {
+                "value": "Charles",
+                "language": null,
+                "authority": null,
+                "confidence": -1,
+                "place": -1
+              }
+            ]
+        },
+        "type": "metadataSuggestionEntry",
+        "_links": {
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid/entryValues/0000-0003-3681-2038"
+          },
+         "changes": {
+           "href": "https://dspace7-internal.atmire.com/server/api/integration/metadata-suggestions/orcid/entryValues/0000-0003-3681-2038/changes"
+         }
+        }
+      }
+    ]
+  }
+}
+```
+
+The changes are explained below, and won't be embedded in this endpoint.
+
+### Single entry
+**/api/integration/metadata-suggestions/<:suggestion-name>/entryValues/<:entry-id>**  
+
+Parameters:
+* workspaceitem: the current workspace item ID (mutually exclusive with workflowitem)
+* workflowitem: the current workflow item ID (mutually exclusive with workspaceitem)
```

Comment:
I'm not sure I understand why `workspaceitem` and `workflowitem` are parameters on this endpoint.  How do they affect the results returned for this single entry?

Reply:
For a single entry, they will only impact the embedded `changes` section.
If the current workspace item already has a dc.title, the `changes` will not suggest to add a second title if this field is not repeatable. It will check the item metadata and the submission forms to suggest relevant changes.
This is complexity we don't want to push to the UI, but rather want to compute in REST

Reply:
@benbosman : minor note, I think we can now remove `workspaceitem` and `workflowitem` from this `/entryValues` endpoint as parameters. They are only now applicable to the `/entryValueDifferences` endpoint.

Reply:
I've removed the parameters since the new structure indeed should not use them